packages/@react-spectrum/tag/src/TagGroup.tsx

```diff
@@ -188,7 +188,7 @@ function TagGroup<T extends object>(props: SpectrumTagGroupProps<T>, ref: DOMRef
             {actionLabel && onAction &&
               <ActionButton
                 isQuiet
-                onPress={onAction}
+                onPress={() => onAction()}
```

Comment:
```suggestion
                onPress={() => onAction?.()}
```

Reply:
I know it'd be weird to provide an action and not have a handler for it, but just in case